data-types.md

```diff
@@ -0,0 +1,307 @@
+
+(?) Do we still want `event_geographies`?
+
+(?) Replace `event_location` GeoJSON with Telemetry
+
+(?) Does anyone use `associated_ticket`?
+
+(?) Does anyone use `publication_time`?
+
+### Event Times
+
+Because of the unreliability of device clocks, the Provider is unlikely to know with total confidence what time an event occurred at. However, Providers are responsible for constructing as accurate a timeline as possible. Most importantly, the order of the timestamps for a particular device's events must reflect the Provider's best understanding of the order in which those events occurred.
+
+[Top][toc]
+
+## Telemetry Data
+
+A standard point of vehicle telemetry. References to latitude and longitude imply coordinates encoded in the [WGS 84 (EPSG:4326)](https://en.wikipedia.org/wiki/World_Geodetic_System) standard GPS or GNSS projection expressed as [Decimal Degrees](https://en.wikipedia.org/wiki/Decimal_degrees).
+
+| Field          | Type           | Required/Optional     | Field Description                                            |
+| -------------- | -------------- | --------------------- | ------------------------------------------------------------ |
+| `telemetry_id` | UUID           | Required              | ID used for uniquely-identifying a Telemetry entry |
+| `device_id`    | UUID           | Required              | ID used in [Register](#vehicle---register)                     |
+| `timestamp`    | [Timestamp][ts]| Required              | Date/time that event occurred. Based on GPS or GNSS clock            |
+| `trip_ids`      | UUID[]           | Required              | If telemetry occurred during a trip, the ID of the trip(s).  If not in a trip, `null`.
+| `journey_id`   | UUID           | Required              | If telemetry occurred during a trip, the ID of the journey.  If not in a trip, `null`.
+| `stop_id`      | UUID           | Required if Applicable | Stop that the vehicle is currently located at. Only applicable for _docked_ Micromobility. See [Stops][stops] |
+| `gps`          | Object         | Required              | Telemetry position data                                      |
+| `gps.lat`      | Double         | Required              | Latitude of the location                                     |
+| `gps.lng`      | Double         | Required              | Longitude of the location                                    |
+| `gps.altitude` | Double         | Required if Available | Altitude above mean sea level in meters                      |
+| `gps.heading`  | Double         | Required if Available | Degrees - clockwise starting at 0 degrees at true North      |
+| `gps.speed`    | Float          | Required if Available | Estimated speed in meters / sec as reported by the GPS chipset                                        |
+| `gps.accuracy` | Float          | Required if Available | Horizontal accuracy, in meters                                           |
```

Comment:
Can we change this field to 'horizontal_accuracy' and add a similar field for 'vertical_accuracy', per #661?

Reply:
Reupping this comment to change this to 2 separate fields.

Reply:
done